build.gradle.kts

```diff
@@ -164,7 +164,13 @@ subprojects {
 
   java {
     toolchain {
+      // Some JDK vendors like Oracle OpenJDK have problems in building trino-connector.
+      // So we will use AMAZON OpenJDK for trino-connector.
       if (project.name == "trino-connector") {
+        val osName = System.getProperty("os.name").lowercase(Locale.getDefault())
+        if (osName.contains("mac")) {
```

Comment:
You can use `OperatingSystem.current().isMacOsX()`.

Also, please add to the how-to-build document to describe our current limitations.

Reply:
> OperatingSystem.current().isMacOsX()

No, I have tried.